scalajvm/app/views/index.scala.html

```diff
@@ -24,30 +24,40 @@
 }
 
 @menu = {
-    <li>
-        Сортировка: @renderOrdering(QuoteOrdering.Time, "по дате")
-        | @renderOrdering(QuoteOrdering.Rating, "по рейтингу")
-    </li>
-    <li>
-        Фильтр: @renderFilter(QuoteFilter.None, "нет")
-        | @renderFilter(QuoteFilter.Year, "за год")
-        | @renderFilter(QuoteFilter.Month, "за месяц")
-        | @renderFilter(QuoteFilter.Week, "за неделю")
-        | @renderFilter(QuoteFilter.Day, "за день")
-    </li>
+    <div class="spaced space-around">
+        <span>Сортировка:&nbsp;</span>
+        <div class="spaced space-around">
+            <span> @renderOrdering(QuoteOrdering.Time, "по дате")</span>
+            <span>&nbsp;| @renderOrdering(QuoteOrdering.Rating, "по рейтингу")</span>
+        </div>
+    </div>
+    <div class="spaced space-around">
+        <span>Фильтр:&nbsp;</span>
+        <div class="spaced space-around">
+            <span>  @renderFilter(QuoteFilter.None, "нет")</span>
+            <span>&nbsp;| @renderFilter(QuoteFilter.Year, "за год")</span>
+            <span>&nbsp;| @renderFilter(QuoteFilter.Month, "за месяц")</span>
+            <span>&nbsp;| @renderFilter(QuoteFilter.Week, "за неделю")</span>
+            <span>&nbsp;| @renderFilter(QuoteFilter.Day, "за день")</span>
+        </div>
+    </div>
 }
 
 @main("Log List", menu) {
     <div class="center-content quotes">
     @quotes.map { quote =>
         <div class="quote">
-            <div class="quote-header">
-                <a href="@routes.Quotes.quote(quote.id)">#@quote.id</a>
-                @voting.ratingPanel(quote)
+            <div class="spaced quote-header">
+                <div>
+                    <span><a href="@routes.Quotes.quote(quote.id)">#@quote.id</a></span>
+                    <div class="desktop-only">@voting.ratingPanel(quote)</div>
+                </div>
                 <span class="quote-time">@quote.time</span>
-                <span class="clear"></span>
             </div>
             <div class="quote-content plate">@quote.content.getOrElse("")</div>
+            <div id="mobile-rating-panel" class="mobile-only plate">
```

Comment:
You're assigning this `id` inside of a `@quotes.map` loop. According to my understanding, it will create multiple elements with the same `id` in a resulting markup. This is invalid; please replace it with a `class`.